setup.cfg

```diff
@@ -1,39 +1,48 @@
 [bumpversion]
-current_version = 0.1.0
-commit = True
-tag = True
+current_version = 0.1.4-beta
+commit = False
+tag = False
+parse = (?P<major>\d+)\.(?P<minor>\d+).(?P<patch>\d+)(\-(?P<release>[a-z]+))?
+serialize = 
+	{major}.{minor}.{patch}-{release}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+optional_value = gamma
+values = 
+	beta
+	gamma
 
 [bumpversion:file:setup.py]
-search = version='{current_version}'
-replace = version='{new_version}'
+search = version="{current_version}"
+replace = version="{new_version}"
 
-[bumpversion:file:ravenpy/__init__.py]
-search = __version__ = '{current_version}'
-replace = __version__ = '{new_version}'
+[bumpversion:file:ravenpy/__version__.py]
+search = __version__ = "{current_version}"
+replace = __version__ = "{new_version}"
 
 [bumpversion:file:.cruft.json]
 search = "version": "{current_version}",
 replace = "version": "{new_version}",
 
 [aliases]
-# Define setup.py command aliases here
 test = pytest
 
 [tool:pytest]
 collect_ignore = ['setup.py']
 addopts = --verbose
-filterwarnings =
+filterwarnings = 
```

Comment:
Your editor seems to be adding an extra space here? Mine is configured to remove it..

Reply:
Oh, it's a strange bug with bump2version. It's harmless so long as we don't commit on bump. https://github.com/c4urself/bump2version/issues/124